cmd/node-labels-resources/doc.go

```diff
@@ -0,0 +1 @@
+package main
```

Comment:
```suggestion
// package main contains the main function for the node-label-resource-plugin.
package main
```

Here you can fix te linter with something like this